modules/configuration/src/main/java/org/apache/ignite/internal/configuration/tree/InnerNode.java

```diff
@@ -187,4 +188,18 @@ public InnerNode copy() {
     public <NODET> NODET specificNode() {
         return (NODET) this;
     }
+
+    /**
+     * Gets the value of a field with {@link InjectedName}.
```

Comment:
"Returns", not "Gets". Andrey Gura is very upset every time he sees the word "gets" in javadoc. Don't upset him :)

Reply:
I hope he doesn't feel bad now.